docs/config-presets-and-options.md

```diff
@@ -0,0 +1,70 @@
+|-----------------------|--------------|--------------|---------------------|
+| High Availablity (HA) |       3+     |       2+     |         No          |
+| Compact               |       3      |       0      |         Yes         |
+| Single Node Openshift |       1      |       0      |         Yes         |
+
+
+### Resources Used for Openshift Installation Validations in the Lab
+
+|                     | High Availablity (HA) | Compact | Single Node OpenShift (SNO) |
+|---------------------|-----------------------|---------|-----------------------------|
+| NUM_MASTERS         |           3+          |    3    |              1              |
+| NUM_WORKERS         |           2+          |    0    |              0              |
+| NUM_EXTRA_WORKERS   |           0           |    0    |              0              |
+| MASTER_MEMORY       |         16384         |  16384  |            32768            |
+| MASTER_DISK         |           20          |    20   |              20             |
+| MASTER_VCPU         |           8           |    8    |              8              |
+| WORKER_MEMORY       |         16384         |         |                             |
+| WORKER_DISK         |           20          |         |                             |
+| WORKER_VCPU         |           4           |         |                             |
+| EXTRA_WORKER_MEMORY |         16384         |         |                             |
+| EXTRA_WORKER_DISK   |           20          |         |                             |
+| EXTRA_WORKER_VCPU   |           4           |         |                             |
+
+
+### Minimum Required Resources for Production Openshift Clusters
+
+|                     | High Availablity (HA) | Compact | Single Node OpenShift (SNO) |
+|---------------------|-----------------------|---------|-----------------------------|
+| NUM_MASTERS         |           3+          |    3    |              1              |
+| NUM_WORKERS         |           2+          |    0    |              0              |
+| NUM_EXTRA_WORKERS   |           0           |    0    |              0              |
+| MASTER_MEMORY       |         16384         |  16384  |            65536            |
+| MASTER_DISK         |          120          |   120   |             120             |
+| MASTER_VCPU         |           8           |    8    |              8              |
+| WORKER_MEMORY       |         16384         |         |                             |
+| WORKER_DISK         |          120          |         |                             |
+| WORKER_VCPU         |           4           |         |                             |
+| EXTRA_WORKER_MEMORY |         16384         |         |                             |
+| EXTRA_WORKER_DISK   |          120          |         |                             |
+| EXTRA_WORKER_VCPU   |           4           |         |                             |
+
+## Defining the Network Topology
```

Comment:
There could be potentially another dimension to be considered here: static ips vs dhcp.
Specifically to the agent installer, currently we're [assigning static ips](https://github.com/openshift-metal3/dev-scripts/blob/4e93a89945d05e9fd91764b1d045c173f3db58b9/agent/03_agent_deployment.sh#L34-L56) starting from 192.168.111.80

Reply:
Just to understand does this mean we don't have the implementation in the rest of dev-scripts for Static IP vs DHCP? 🤔

Reply:
There's isn't any specific for that, even though that are a couple of PRs in progress to inject a static ip in the bootstrap